ui/root/src/commonMain/kotlin/social/androiddev/root/splash/SplashContent.kt

```diff
@@ -60,12 +60,12 @@ fun SplashContent(
         Text("Loading")
 
         LaunchedEffect(authStatus) {
-            if (authStatus is AuthStatus.Authorized) {
```

Comment:
We should always prefer exhaustive `when` over if/else chaining when using enum or sealed class

Reply:
I'm only interested in 2 cases out of 3, does the rule still apply?

Reply:
Still good practice, so you get the compiler warning/error if you ever add a new value to the enum/sealed class. You can always have it just be a no-op here for the ones you don't care about